test/test_ssl.py

```diff
@@ -149,3 +149,18 @@ def test_wrap_socket_default_loads_default_certs(monkeypatch):
     ssl_.ssl_wrap_socket(sock)
 
     context.load_default_certs.assert_called_with()
+
+
+@pytest.mark.parametrize(
+    ["pha", "expected_pha"], [(None, None), (False, True), (True, True)]
```

Comment:
@tiran We were wondering with @jab in https://github.com/python-trio/urllib3/pull/127: any reason why setting PHA to False sets it back to True? In other words, why `(False, True)` and not `(False, False)`?